jetty-p2/pom.xml

```diff
@@ -0,0 +1,50 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
nit. we usually a 2 spaces indentation in pom files

Reply:
As it seems you have added a change to this already (thanks for that) is this addressed or do I need to do anything here? Is there any eclipse-prefrences/formatters I can import/use for the jetty project?

Reply:
yes please use 2 spaces indentation for pom.xml